### ISSUE TITLE
Direct big datasette downloads to AWS Open Data Instead

### ISSUE THREAD

**Issue by zaneselvans** [catalyst-cooperative/pudl#2464]
Given that we've just tripled the size of the PUDL DB, we may want to be a little forceful about directing folks to do bulk downloads from the AWS Open Data Catalog. Some clear options:

```[tasklist]
- [ ] Remove direct DB download or point at AWS instead.
- [ ] Disable CSV streaming.
- [ ] Add prominent links to AWS on each DB index page / top level index.
```

**Comment by ggurjar333** [comment]
Can I TAKE this?

**Comment by bendnorman** [comment]
Yes! I think this could be a good first issue. It looks like the links to download the full databases have been removed.

To fix this issue I think you'll need to:
1. Turn off CSV streaming by adding [`allow_csv_stream off`](https://docs.datasette.io/en/stable/settings.html#setting-allow-csv-stream) to the `datasette` command in the `devtools/datasette/fly/run.sh` script.
2. Add [s3 download links](https://catalystcoop-pudl.readthedocs.io/en/dev/data_access.html#nightly-builds) to the database descriptions. To update the descriptions, edit the `src/pudl/metadata/templates/datasette-metadata.yml`.

**Comment by bendnorman** [comment]
I don't think it's worth doing this now that we are moving towards our own user portal and the parquet files are available in the data dictionary.
